src/NuGet.Clients/NuGet.PackageManagement.UI/Utility/WpfUtility.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Windows;
+using System.Windows.Media;
+
+namespace NuGet.PackageManagement.UI
+{
+    public static class WpfUtility
+    {
+        public static T FindParent<T>(this DependencyObject element)
```

Comment:
I think I like the name 'FindAncestor' better for this method. Unless there's any objection, I'll change the name.

Reply:
Could we use this or similar Extension method instead of creating our own? [Microsoft.VisualStudio.PlatformUI : ExtensionMethods.FindAncestor](https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.platformui.extensionmethods.findancestor?view=visualstudiosdk-2019)

Reply:
I see this was marked resolved, but would appreciate a response to my question either way please.

Reply:
Good find! I switched to using the FindAncestor extension method from PlatformUI and removed WpfUtility altogether.

FYI: Your comment wasn't showing for me when I resolved this thread and only my comment was. I'm noticing sometimes the comments don't get automatically refreshed if the PR page is open for a while.